pkg/istio-agent/sds-agent.go

```diff
@@ -430,6 +430,9 @@ func applyEnvVars() {
 	workloadSdsCacheOptions.SecretTTL = secretTTLEnv
 	workloadSdsCacheOptions.SecretRefreshGraceDuration = secretRefreshGraceDurationEnv
 	workloadSdsCacheOptions.RotationInterval = secretRotationIntervalEnv
+	// If the secret is not rotated for 3*workloadSdsCacheOptions.SecretTTL,
+	// it is regarded as stale and will be evicted.
+	workloadSdsCacheOptions.EvictionDuration = 3 * workloadSdsCacheOptions.SecretTTL
```

Comment:
This option is pretty scary... basically if we ever miss a cert rotation for whatever reason and we end up hitting this when we shouldn't, we are going to be permanently broken and never rotate certs? Why do we even need this for workload, there is only even `default` and `ROOTCA`?

Reply:
I don't think we need this eviction functionality for istio agent, since unlike node agent, it's only serving a single sidecar.